frontend/routes/components/topMenu/components/visualSettingsDialog/visualSettingsDialog.styles.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ *  Copyright (C) 2019 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+import { COLOR } from '../../../../../styles';
+import { Button, DialogContent, ListItem, Tooltip, Input, Tabs, Tab } from '@material-ui/core';
+import * as React from 'react';
+import { TooltipProps } from '@material-ui/core/Tooltip';
+import {omit} from 'lodash';
+
+export const NegativeActionButton = styled(Button)`
+  && {
+    color: ${COLOR.WHITE_87};
+	background-color:  rgba(234, 57, 57, 0.87);
+  }
+
+  &&:hover {
+	background-color:  rgba(234, 57, 57, 1);
+  }
+`;
+
+export const NeutralActionButton = styled(Button)`
+  && {
+    color: ${COLOR.BLACK_60};
+	background-color:  rgba(256, 256, 256, 0);
```

Comment:
Do you see these extra spaces too? If yes, please change it. If not, it's probably github bug...  
![image](https://user-images.githubusercontent.com/5683490/54017139-6572f800-4185-11e9-9abd-219eb9859b77.png)

Reply:
Yest, its because of the mixture between tabs and spaces. Its easy to mix up both tabbing styles (the one from the general ts and the one for the strings in the styled components ts).

Reply:
linter has indent only rule enforced on tslint. Why are these getting through the check? 😮

Reply:
Styles are checked using stylelint, so tslint config doesn't affect checking them.
I've checked styleint config and there is no indentation rule. I'll fix it. Do you want these changes here or can I provide it on e.g Issues PR?

Reply:
Yea Issues PR will be great :)